gateway/metrics/metrics.go

```diff
@@ -44,16 +52,40 @@ func BuildMetricsOptions() MetricOptions {
 		[]string{"function_name"},
 	)
 
+	// For automatic monitoring and alerting (RED method)
+	histogram := prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Subsystem: "http",
+		Name:      "request_duration_seconds",
+		Help:      "Seconds spent serving HTTP requests.",
+		Buckets:   prometheus.DefBuckets,
+	}, []string{"method", "path", "status"})
+
+	// Can be used Kubernetes HPA v2
+	counter := prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Subsystem: "http",
+			Name:      "requests_total",
+			Help:      "The total number of HTTP requests.",
+		},
+		[]string{"status"},
```

Comment:
should this metric also have method and path too?

Reply:
I took the example from @stefanprodan as "what Kubernetes/managed dashboards mandates" and asked the same question on Slack. I don't know.

@stefanprodan can other bits of data be recorded here too and still be compatible?

Reply:
Compatibility is going to depend on which "managed dashboard" we want to support, making an explicit list will help and it would be good to comment about those in the code to ensure future changes don't accidentally break it.

In my experience with prometheus queries, additional labels shouldn't be an issue as long as the required labels exist.

Reply:
Personally I'd like the same labels as on the histogram at least.

Reply:
agreed, I was confused and thought I had the wrong image when I first queried the metrics

Reply:
Added labels to match both the counter and histogram